auth/downscoping/snippets_test.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+import tempfile
+
+import pytest
+
+from snippets import main
+
+
+BUCKET = os.environ["CLOUD_STORAGE_BUCKET"]
```

Comment:
Another option is to create and tear down the bucket and blob in the test.

See this example: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/33e3de913c30d430acc040094264b36142c04d89/storage/s3-sdk/s3_sdk_test.py#L47-L65

Reply:
Done.